src/app/input-validator/input-validator.component.html

```diff
@@ -1,12 +1,12 @@
 @if (property) {
-  @if ((this.property?.invalid && this.property?.touched) ?? false) {
+  @if ((this.property.invalid && this.property.touched) ?? false) {
```

Comment:
the defautl value after ?? is not needed anymore as you check for null values in the above if statement